### PR TITLE
feat: Back up wallet page - components

### PR DESCRIPTION
**Pull request by andrzejchm**

### CODE REVIEW

File: packages/cosmos_ui_components/lib/components/mnemonic_choice_chip.dart [hunk 1]
```diff
@@ -17,42 +18,54 @@ class MnemonicChoiceChip extends StatelessWidget {
...
         ),
       ),
     );
   }
+
+  Widget _indexText(BuildContext context, int index) => Padding(
+        padding: EdgeInsets.only(right: CosmosTheme.of(context).spacingM),
+        child: Container(
+          width: 18,
+          height: 18,
```
Comment (wal33d006) [inline packages/cosmos_ui_components/lib/components/mnemonic_choice_chip.dart:56]: Do we have a special requirement of an 18x18 text box?

**Review by wal33d006 — APPROVED** [review]
Just one comment, other than that good to go!

File: packages/cosmos_ui_components/lib/components/mnemonic_choice_chip.dart (hunk 1 above)
Comment (andrzejchm) [reply packages/cosmos_ui_components/lib/components/mnemonic_choice_chip.dart:56]: its according to the figma design:
image

Comment (wal33d006) [reply packages/cosmos_ui_components/lib/components/mnemonic_choice_chip.dart:56]: Perfect

**Pull request merged**